cbmc_viewer/optionst.py

```diff
@@ -114,6 +114,22 @@ def property(parser):
     )
     return parser
 
+def byteop(parser):
+    'Define --byteop command line option.'
+
+    parser.add_argument(
+        '--byteop',
+        metavar='FILE',
+        default='byteops.json',
```

Comment:
Should this flag have a default value?  Doesn't this assume that cbmc-viewer will ALWAYS be run with this metic information available?  This is certainly not true.